pttrack/forms.py

```diff
@@ -86,6 +86,28 @@ class Meta:
         model = models.Patient
         exclude = []
 
+    def clean(self):
+
+        cleaned_data = super(ModelForm, self).clean()
+
+        alternate_phone_list = ["alternate_phone_1", "alternate_phone_1_owner","alternate_phone_2", "alternate_phone_2_owner",
+                                "alternate_phone_3", "alternate_phone_3_owner","alternate_phone_4", "alternate_phone_4_owner"]
+
+        for i in [0, 2, 4, 6]:
```

Comment:
This isn't very important, but the pythonic way of doing this would be to use the [range](https://docs.python.org/2/library/functions.html#range) built-in. So, you might have something like

```python
    for i in range(0, 7, 2)
```

Because you want to start at 0, end at 6 (seven because it'll stop at 5 if you put 6), and go by twos.